src/core/lib/security/credentials/credentials.cc

```diff
@@ -45,6 +45,14 @@ void grpc_channel_credentials_release(grpc_channel_credentials* creds) {
   if (creds) creds->Unref();
 }
 
+grpc_channel_credentials* grpc_channel_credentials_copy(
+    grpc_channel_credentials* creds) {
+  GRPC_API_TRACE("grpc_channel_credentials_copy(creds=%p)", 1, (creds));
+  grpc_core::ExecCtx exec_ctx;
+  if (creds) creds->Ref().release();
```

Comment:
It is odd to return the original creds.
You can either make the function void or return the copied object, e.g., 
` if (creds) return creds->Ref().release();`

Reply:
This is to be consistent with other similar functions, like channelz_node_copy() and target_authority_table_copy().